Packet++/header/GtpLayer.h

```diff
@@ -211,243 +211,244 @@ namespace pcpp
 	 * @class GtpV1Layer
 	 * A class representing the [GTP v1](https://en.wikipedia.org/wiki/GPRS_Tunnelling_Protocol) protocol.
 	 */
-    class GtpV1Layer : public Layer
-    {
-	private:
-		struct gtpv1_header_extra
+		class GtpV1Layer : public Layer
```

Comment:
The tabs indentation of this class is not aligned with the rest of the file

Reply:
Done